components/rms/slurm/SPECS/slurm.spec

```diff
@@ -152,6 +152,11 @@ BuildRequires: pkgconfig
 %endif
 %endif
 
+%if 0%{?openEuler}
+BuildRequires: perl perl-devel
+Requires: perl perl-devel
```

Comment:
This line looks unnecessary. The is usually picked up automatically

Reply:
Removed the `Requires`!